pom.xml

```diff
@@ -604,7 +604,7 @@
       <dependency>
         <groupId>org.slf4j</groupId>
         <artifactId>slf4j-api</artifactId>
-        <version>2.0.9</version>
+        <version>1.7.25</version>
```

Comment:
We cannot change slf4j, otherwise spart 3.5 will fail.

My suggestion was to try to use slf4j2 v2.0.x and `log4j-slf4j18-impl` as adaptor, although I'm not sure if that would work

Reply:
that is what i had tried in my first commit, but got the error:
```
java.lang.abstractmethoderror: receiver class org.apache.logging.slf4j.slf4jserviceprovider does not define or inherit an implementation of the resolved method 'abstract java.lang.string getrequestedapiversion()' of interface org.slf4j.spi.slf4jserviceprovider.
```

presumably related to https://issues.apache.org/jira/browse/LOG4J2-3370